packages/wallet/src/keyrings/HDKeyring.spec.ts

```diff
@@ -18,42 +18,42 @@ import {IKeyring} from '@plugnet/wallet/types';
 
 import {HDKeyring} from './HDKeyring';
 
-describe('HDKeyring', () => {
-    beforeAll(async () => {
-        await cryptoWaitReady();
-    });
-
-    describe('happy path', () => {
-        const times = 10;
-        let keyring: IKeyring<any>;
-        let keyPairs: KeyringPair[];
-        beforeEach(async () => {
-            keyring = await HDKeyring.generate();
-            keyPairs = [];
-            for (let i = 0; i < times; i++) {
-                const keyPair = await keyring.addPair();
-                keyPairs.push(keyPair);
-            }
-            expect(keyPairs).toHaveLength(times);
-        });
+describe.skip('HDKeyring', () => {
```

Comment:
Does it mean wallet is not working? We may need a ticket accordingly and prioritise it later?

Reply:
we have a ticket for fixing wallet.. I will add a comment to the link to ticket?

Reply:
gotcha, thanks